packages/@aws-cdk/cloudformation-include/test/invalid-templates.test.ts

```diff
@@ -100,6 +100,24 @@ describe('CDK Include', () => {
       includeTestTemplate(stack, 'output-referencing-nonexistant-condition.json');
     }).toThrow(/Output with name 'SomeOutput' refers to a Condition with name 'NonexistantCondition' which was not found in this template/);
   });
+
+  test("throws a validation exception when Fn::Sub in string form uses a key that isn't in the template", () => {
+    expect(() => {
+      includeTestTemplate(stack, 'fn-sub-invalid-key-string.json');
```

Comment:
The names of these are too generic. How is it invalid? In this case, it's a missing/non-existent resource. The name of the file should reflect that.